server/data/exampleApiClient.test.ts

```diff
@@ -1,23 +1,24 @@
+import { beforeEach, afterEach, Mocked, vi, expect, it, describe } from 'vitest'
```

Comment:
By default vite requires explicit import but you can configure it to use globals which might make a migration more easy.. I hit linting/tsc issues which probably can be overcome.